main.go

```diff
@@ -46,7 +46,12 @@ func main() {
 	fmt.Printf("\nExecuting Matrix...\n\n")
 	results := execute.Run(plan)
 
-	summary := output.Stream(results)
+	reporter, err := output.GetReporter(config.Report)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
Assuming that the output has to go to stdout? If not, `log.Fatalln` will do this for you.